model/pdata/generated_metrics.go

```diff
@@ -1233,12 +1233,14 @@ func (ms DoubleDataPoint) SetTimestamp(v Timestamp) {
 
 // Value returns the value associated with this DoubleDataPoint.
 func (ms DoubleDataPoint) Value() float64 {
-	return (*ms.orig).Value
+	return (*ms.orig).GetAsDouble()
 }
```

Comment:
Do we plan to remove `Value` and expose 2 different types of values, the `int` and the `double` in a future PR?

Reply:
Yes